src/communication/dummy/dummy_communicator_backend.cpp

```diff
@@ -0,0 +1,72 @@
+/***************************************************************************
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+/**
+ * @file dummy_communicator_backend.cpp
+ * @author Oier Lauzirika Zarrabeitia (oierlauzi@bizkaia.eu)
+ * @brief Implementation of dummy_communicator_backend.hpp
+ * @date 2024-10-26
+ * 
+ */
+
+#include <xmipp4/core/communication/dummy/dummy_communicator_backend.hpp>
+
+#include <xmipp4/core/communication/dummy/dummy_communicator.hpp>
+#include <xmipp4/core/communication/communicator_manager.hpp>
+
+#include <xmipp4/core/core_version.hpp>
+
+namespace xmipp4 
+{
+namespace communication
+{
+
+static const std::string name = "dummy";
+
+const std::string& dummy_communicator_backend::get_name() const noexcept
+{
+    return name;
+}
```

Comment:
Why define name and then return it as variable.

This was going to change in #98, so, i'd recommend following the same path here, as that PR will be merged really soon.
(alternatively, we could separate that into a different PR and follow it's own merge process in every repo)

Reply:
It is like that so that it compiles in the action (the changes from #98 are not merged). Depending on which one is merged before, it will be changed in this PR or in the other